refinery/data_set_manager/utils.py

```diff
@@ -806,10 +800,17 @@ def format_solr_response(solr_response):
         solr_response_json = json.loads(solr_response)
     except TypeError:
         return "Error loading json."
+    assert solr_response_json['responseHeader']['status'] == 0, \
```

Comment:
@mccalluc It may be useful to return HTTP 500 response in a case like this but exceptions should not be left unhandled: "Care must be taken when handling exceptions to ensure proper application cleanup while maintaining useful error reporting." https://doughellmann.com/blog/2009/06/19/python-exception-handling-techniques/